docs/download_files.md

```diff
@@ -95,32 +95,51 @@ Every download also includes the individual [QC report](#qc-report) and, if appl
+All spatial data for any samples sequenced using [spatial transcriptomics](#spatial-transcriptomics-libraries) is available separately as a zip file. 
 
-### `SingleCellExperiment` downloads
+When downloading any of the available Portal-wide data downloads all metadata and associated bulk RNA-seq data is also included. 
 
-#### Download folder structure for individual sample downloads:
-![sample download folder](images/sample-download-folder.png){width="600"}
+<!--TODO: make sure data format naming for files is correctly formatted-->
+Each zip file will be named with the the chosen data format (either `SingleCellExperiment` or `AnnData`), and the date you accessed the data on the ScPCA Portal.
+Within each zip file will be a folder for each project that contains the gene expression data for all samples in that project as either individual objects or a single [merged object](#portal-wide-downloads-as-merged-objects). 
 
-### `AnnData` downloads
+As with [individual project](#project-downloads) and [custom datasets](#custom-datasets), the quantified CITE-seq expression data will be included when downloading single-cell expression data.
+For [`SingleCellExperiment (R)` downloads](#singlecellexperiment-portal-wide-download-structure), the quantified CITE-seq expression is included in the same file as the gene expression data. 
+For [`AnnData (Python)` downloads](#anndata-portal-wide-download-structure), the quantified CITE-seq expression data is included as a separate file with the suffix `_adt.h5ad`.
 
-#### Download folder structure for individual sample downloads:
-![sample download folder](images/anndata-sample-download-folder.png){width="600"}
+<!--TODO: Update images-->
+### `SingleCellExperiment` Portal-wide download structure
+![portal wide download structure - `sce`](STUB-IMAGE){width="600"}
 
-#### Download folder structure for individual sample downloads with CITE-seq (ADT) data:
-![sample download folder](images/anndata-sample-citeseq-download-folder.png){width="600"}
-
-If downloading a sample that contains a CITE-seq library as an `AnnData` object (`.h5ad` file), the quantified CITE-seq expression data is included as a separate file with the suffix `_adt.h5ad`.
+### `AnnData` Portal-wide download structure
+![portal wide download structure - `anndata`](STUB-IMAGE){width="600"}
 
 <!--TODO:
-spatial section to be moved here, followed by the merged objects section
+spatial section to be moved here
 -->
 
+### Portal wide downloads as merged objects
+
+You can choose to download data from all samples as [merged objects with data from all samples in a single project](#merged-object-downloads) by checking "Merge samples into one object per project". 
```

Comment:
I'm suggesting splitting some of these details up, what do you think? Mostly I found the phrasing "all samples as merged objects with data from all samples" confusing, so if you don't like this change that's all good but I would want to see that phrasing updated.

Also, we should be clear this is is only single-cell (nuclei) data in the merged. 
```suggestion
You can choose to download single-cell Portal data as [merged objects for each project](#merged-object-downloads) by checking "Merge samples into one object per project". 
{ref}`Merged objects<merged_objects:Merged objects>` contain gene expression for all samples in a given project in a single file.
```